docs/sphinx/sections/api/apidocs/asset-checks.rst

```diff
@@ -0,0 +1,14 @@
+.. currentmodule:: dagster
+
+Asset Checks
+============
+
+Asset Checks are functions that validate some property of a software-defined asset.
```

Comment:
Not every asset check is its own function. The way I kind of got around this in the preliminary concept docs was with "Each asset check verifies some property of a data asset, e.g. that is has no null values in a particular column."